packages/block-editor/src/components/use-block-drop-zone/index.js

```diff
@@ -274,6 +274,27 @@ export function isDropTargetValid(
 	return areBlocksAllowed && targetMatchesDraggedBlockParents;
 }
 
+/**
+ * Check if the Row or the Gallery can be created with the dragged
+ * blocks and the target block.
+ * @param {boolean} areAllImages
+ * @param {boolean} canInsertGalleryBlock
+ * @param {boolean} areGroupableBlocks
+ * @param {boolean} canInsertRow
+ * @return {boolean} Whether Row block or Gallery can be created.
+ */
+function canCreateRowOrGallery(
```

Comment:
This process might not even need to be made into a function.

Reply:
Same. Since it's only used in one place, I'd lean slightly toward inlining unless it needs unit testing.

Reply:
I removed the function and inlined it [here](https://github.com/WordPress/gutenberg/pull/64241/files#diff-43d4fac82b9f9255055e90135a8dfe36f59a2c9de24b731dae8b9d5fcd3581b1R433-R449) 👍